docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -319,14 +319,14 @@ you should do those steps:
 
 .. code-block:: docker
 
-    #image: ${AIRFLOW_IMAGE_NAME:-apache/airflow:2.6.1}
+    #image: ${AIRFLOW_IMAGE_NAME:-apache/airflow:2.8.0}
```

Comment:
And here we can use: 
```suggestion
    #image: ${AIRFLOW_IMAGE_NAME:-apache/airflow:<airflow version>}
```

Reply:
We have `|version|` that replaces Airflow version in generated documentation. Probably that's best approach.

Reply:
It might require some manual modifications in doc build scripts in some cass, but I think it will just work here out-of-the box, you can try it and see if it gets replaced after `breeze build-docs` build. There is a sphinx plugin we use to do it I think.

If there are any problems - look for `|version|` - there are a few places where it is replaced by hand during the build.